### DISCUSSION TITLE
Turn off back window LED on z13

### DISCUSSION THREAD

**Discussion by PxSigma** [seerge/g-helper#1482] [Ideas]
Thats it. It is kinda hidden on armoury crate, but available. Lets you customize the back window led when you are plugged or not. It reduces some of the power consuption, which is great given that the battery life isn't that awesome on the tablet. I triied to use both but ghelper competes with armoury crate and keeps the back led turned on. If this feature could be added, it would be great.

**Comment by seerge** [comment]
@PxSigma hello, you can already do that with ``Awake`` backlight checkboxes under ``Extra`` :)

**Reply by PxSigma** [reply]
I tried and it blinks if I choose any of the options, but it continues on even if I leave it unmarked. Is there something special I should do? I have already uninstalled armoury crate again and stopped the asus apps via ghelper.

**Reply by seerge** [reply]
Do you have any leftovers from AC running by any chance ? Like Aura service ? it may be setting lights as well

**Reply by PxSigma** [reply]
Just checked, everything is stopped and/or uninstalled. The back window light still blinks when I choose any of the options. Is it limited for the 2023 z13? And is it on the lid or logo option?

**Reply by seerge** [reply]
@PxSigma i think it's logo option :) thing is that i was explicitly adding support for backlight window there for z13 2023, and it was confirmed by multiple users to be working.

I don't really know what is your specific reason.

**Reply by PxSigma** [reply]
Probably just my luck, but my version is the 2022 i9 3050ti. I will tinker some more and try to come with answers if i succeed

**Reply by seerge** [reply]
@PxSigma check if OpenRGB is able to reach and control backlight window on your device

**Reply by PxSigma** [reply]
Hi, I would like to close/mark as solved as it was the very same issue fixed on 0.148 after #1930
